Unix/regress

```diff
@@ -182,25 +181,26 @@ fi
 ##
 ## 'wchar' tests:
 ##
+## wchar has been deprecated
 ##==============================================================================
-
-if [ "$run_wchar" = "1" ]; then
-
-current_output=output2
-sudo rm -rf ./$current_output GNUmakefile
-createlink
-$root/configure --dev --enable-wchar --outputdirname=$current_output $options
-make world
-
-if [ "$?" != "0" ]; then
-    echo "$0: failed 'wchar' tests"
-    cleanup  $current_output
-    exit 1
-fi
-
-sudo rm -rf ./$current_output $new_output GNUmakefile
-
-fi
+##
+##if [ "$run_wchar" = "1" ]; then
+##
+##current_output=output2
+##sudo rm -rf ./$current_output GNUmakefile
+##createlink
+##$root/configure --dev --enable-wchar --outputdirname=$current_output $options
+##make world
+##
+##if [ "$?" != "0" ]; then
+##    echo "$0: failed 'wchar' tests"
+##    cleanup  $current_output
+##    exit 1
+##fi
+##
+##sudo rm -rf ./$current_output $new_output GNUmakefile
+##
+##fi
```

Comment:
Strictly speaking, if `$run_wchar` is always zero, you don't need to comment this out at all. But I still recommend removing it to keep it clean, shorter, and thus easier to understand.